sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/ConvertHelpers.java

```diff
@@ -85,57 +100,58 @@ public ConvertedSchemaInformation(
   public static <T> ConvertedSchemaInformation<T> getConvertedSchemaInformation(
       Schema inputSchema, TypeDescriptor<T> outputType, SchemaRegistry schemaRegistry) {
 
-    ConvertedSchemaInformation<T> schemaInformation = null;
+    ConvertedSchemaInformation<T> schemaInformation;
```

Comment:
nit: move this definition to where assigned

Reply:
In practice, new classes would have to be loaded / added to classpath. I think the way we use service loaders is fixed when things are launched, unless someone is getting quite fancy.

Reply:
Fixed